nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -226,12 +226,10 @@ BtNavigator::navigateToPose()
 
     case nav2_behavior_tree::BtStatus::FAILED:
       RCLCPP_ERROR(get_logger(), "Navigation failed");
-      action_server_->terminate_goals();
```

Comment:
I would like to have an explicit termination of the goal in the callback. Removing this also removes the ability to do simple action server results. For instance, I can no longer have a feedback portion of the action massage when using the simple action server, therefore I can't tell the caller why it was terminated, where it was terminated, or any other things relevant to the action's decision to end.

Reply:
Good points, I'll add the termination back.